spring-cloud-stream-binders/spring-cloud-stream-binder-rabbit/src/main/java/org/springframework/cloud/stream/binder/rabbit/RabbitMessageChannelBinder.java

```diff
@@ -232,6 +232,8 @@ public MessageProperties toMessageProperties(AMQP.BasicProperties source, Envelo
 
 	private volatile int defaultTxSize = DEFAULT_TX_SIZE;
 
+	protected volatile boolean defaultDurableSubscription = true;
+
```

Comment:
The goal is to find the right defaults before the RC. To me, it seems natural that anonymous consumers will be non-durable by default (eg. bus consumers are unaffected), whereas named/grouped will be durable by default (i.e. will reconnect where they left), and Rabbit users have the option of turning that off (not every other binder does). I could be wrong though, and I'm happy to revert that.

But I am trying to understand your stance on demoting durability properties. The notion of ignoring global options is not consistent with https://github.com/spring-cloud/spring-cloud-stream/pull/357#issuecomment-185267078. Are you suggesting that we switch the validation process to property values, instead? I'm fine with that, too, as long as we don't keep going back and forth on the validation process.